doc/source/skips/1-governance.rst

```diff
@@ -0,0 +1,165 @@
+governance model, and make decisions about funding granted to the project
+itself. (Funding to community members is theirs to pursue and
+manage.) The purpose of the SC is to ensure smooth progress from the
+big-picture perspective. Changes that impact the full project require a
+synthetic analysis and a consensus that is both explicit and informed. When the
+core developer community (including the SC members) fails to reach such a
+consensus in a reasonable timeframe, the SC is the entity to resolve the issue.
+
+Membership of the SC is by nomination by a core developer. A nomination will
+result in discussion that cannot take more than a month and then admission to
+the SC by consensus. SC members who do not actively engage
+with the SC duties are expected to resign.
+
+The initial Steering Council of scikit-image consists of Stéfan van der
+Walt, Juan Nunez-Iglesias, Emmanuelle Gouillart, and Josh Warner.
+
+Decision Making Process
+=======================
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the project `mailing list <mailto:scikit-image@python.org>`_
+and the `issue tracker <https://github.com/scikit-image/scikit-image/issues>`_.
+Occasionally, sensitive discussion may occur on a private list.
+
+Decisions should be made in accordance with the :ref:`mission, vision and
+values <values>` of the scikit-image project.
+
+Scikit-image uses a “consensus seeking” process for making decisions. The group
+tries to find a resolution that has no open objections among core developers.
+If no option can be found without objections, the decision is escalated to the
+SC, which will itself use consensus seeking to come to a resolution. In the
+unlikely event that there is still a deadlock, the proposal will move forward
+if it has the support of a simple majority of the SC. Any vote must be
+backed by a :ref:`scikit-image proposal (SKIP) <skip>`.
+
+Decisions (in addition to adding core developers and SC membership as above)
+are made according to the following rules:
+
+- **Minor documentation changes**, such as typo fixes, or addition / correction of a
+  sentence (but no change of the scikit-image.org landing page or the “about”
+  page), require approval by a core developer *and* no disagreement or requested
+  changes by a core developer on the issue or pull request page (lazy
+  consensus). Core developers are expected to give “reasonable time” to others
+  to give their opinion on the pull request if they’re not confident others
+  would agree.
+
+- **Code changes and major documentation changes** require agreement by *two*
+  core developers *and* no disagreement or requested changes by a core developer
+  on the issue or pull-request page (lazy consensus).
+
+- **Changes to the API principles and changes to dependencies or supported
+  Python versions** require a :ref:`skip` and follow the decision-making
```

Comment:
Too arduous to impose SKIP on version / dependency changes?  Especially now that we may move to a [time-based dependency strategy](https://github.com/numpy/numpy/pull/14086).

Reply:
We should think about what exactly requires a SKIP.  Major intrusive change?

Reply:
I like the time-based idea a lot. =) I've simply removed changes to the dependencies from things that require SKIPs.

Reply:
I like the time based strategy for *version changes* as well. But wouldn't the new phrasing also apply to removing or adding dependencies entirely as well? Is that indented? Shouldn't adding numba as a dependency certainly require a SKIP?

Reply:
You are right. I'll add that back in!

Reply:
> @stefanv also with regards to @rgommers's comments, I think that we are not there yet when it comes to clear roles. I would wait until we have a bit more funding and dedicated time to the project. Currently (most) SC members are volunteers so I don't want to expect much from them other than "jump in when you feel like it". SC membership bears _some_ responsibility, but as I see it, mostly it grants some authority to break deadlocks.

With the exception of the chair, or at least someone who takes responsibility for the structure and membership of the SC.